config/features.yml

```diff
@@ -224,26 +236,18 @@ features:
     actor_type: user
     description: When enabled, claims status tool uses the Lighthouse API for the show endpoint
     enable_in_development: true
-  cst_include_ddl_boa_letters:
-    actor_type: user
-    description: When enabled, the Download Decision Letters feature includes Board of Appeals decision letters
-    enable_in_development: true
-  cst_include_ddl_5103_letters:
+  cst_send_evidence_failure_emails:
```

Comment:
This is the feature toggle I added, the rest of the changes are me alphabetizing all of the feature toggles that begin with the prefix `cst_`

Reply:
I found `cst_include_ddl_boa_letters` and `cst_include_ddl_sqd_letters` in the [v0 claim letters controller](https://github.com/department-of-veterans-affairs/vets-api/blob/master/app/controllers/v0/claim_letters_controller.rb#L45) and in the associated spec file.

I found `cst_synchronous_evidence_uploads` in the [Lighthouse Benefits Service](https://github.com/department-of-veterans-affairs/vets-api/blob/master/lib/lighthouse/benefits_documents/service.rb#L71) and in the associated spec file.

Can you please make another pull request to remove all instances of those 3 flippers?

Reply:
Can I ask why you want them removed? We are still using these feature toggles in production currently

Reply:
Oh wow, sorry for the confusion. It's a habit to search for deleted flippers when reviewing PRs.

Reply:
No worries, sometimes the git diffs get a bit messy. Thanks for the review!